src/libs/TransactionUtils.js

```diff
@@ -120,13 +120,23 @@ function getDescription(transaction) {
  * @returns {Number}
  */
 function getAmount(transaction, isFromExpenseReport) {
-    // In case of expense reports, the amounts are stored using an opposite sign
-    const multiplier = isFromExpenseReport ? -1 : 1;
+    // IOU requests cannot have negative values but they can be stored as negative values, let's return absolute value
+    if (!isFromExpenseReport) {
+        const amount = lodashGet(transaction, 'modifiedAmount', 0);
+        if (amount) {
+            return Math.abs(amount);
+        }
+        return Math.abs(lodashGet(transaction, 'amount', 0));
+    }
+
+    // Expense report case:
+    // The amounts are stored using an opposite sign and negative values can be set,
+    // we need to return an opposite sign than is saved in the transaction object
     const amount = lodashGet(transaction, 'modifiedAmount', 0);
     if (amount) {
-        return multiplier * amount;
+        return -amount;
```

Comment:
This could return `-0`, no?

Reply:
but thats still 0 as a number?

Reply:
amount is not string so its hould be fine

Reply:
From a quick search, it seems like in most cases JS treats them the same. The only difference is in Object identity, i.e. `console.log(Object.is(-0, 0)); // false` so I think we should be ok here